internal/uvm/create.go

```diff
@@ -12,6 +12,7 @@ type Options struct {
 	Owner                   string                  // Specifies the owner. Defaults to executable name.
 	Resources               *specs.WindowsResources // Optional resources for the utility VM. Supports Memory.limit and CPU.Count only currently. // TODO consider extending?
 	AdditionHCSDocumentJSON string                  // Optional additional JSON to merge into the HCS document prior
+	UseGuestConnection      *bool                   // Whether the HCS should connect to the UVM's GCS
```

Comment:
This is LCOW only. Windows without a Guest Connection makes no sense

Reply:
We'd need a guest connection even to test the UVM boot time for WCOW? If so, I'll change it to LCOW only.

Reply:
I cant think of any way we can get output from Windows without a GuestConnection. We do not control the runtime inside the guest

Reply:
Good to know, I'll fix this up.